examples/fullstatus.py

```diff
@@ -2,14 +2,11 @@
 
 from juju.client.connection import Connection
 from juju.client.client import ClientFacade
-
-
-loop = asyncio.get_event_loop()
-conn = loop.run_until_complete(Connection.connect_current())
-
+from juju import loop
 
 async def status():
     client = ClientFacade()
```

Comment:
I didn't realize we had an example exposing the raw API facades.   This should probably be added as a method on the object layer.